starboard/linux/shared/test_filters.py

```diff
@@ -21,7 +21,7 @@
 
 # pylint: disable=line-too-long
 _FILTERED_TESTS = {
-    'nplb': [],
+    'nplb': ['SbMemoryMapTest.CanChangeMemoryProtection'],
```

Comment:
This test crashes when run on a build with coverage enabled. Needs further investigation.

Reply:
We discussed making coverage config a whole individual thing, but if we are looking at just this one test being disabled i think it's fine to go in as is - as long as we have a follow-up bug to deal with it.